src/modelcompiler.cpp

```diff
@@ -143,7 +143,7 @@ void RunCompiler(const std::string &modelName, const std::string &filepath, cons
 	}
 
 	timer.Stop();
-	Output("Compiling \"%s\" took: %lf\n", modelName.c_str(), timer.millicycles());
+	Output("Compiling \"%s\" took: %0.2fms\n", modelName.c_str(), timer.milliseconds());
```

Comment:
This is a MUCH better metric, no idea why I didn't do this originally, thankyou

Reply:
To be fair, I didn't write `Profiler::Clock` until well after I got here. At the time the modelcompiler was written, the only wall-clock timing code in the codebase was a homespun abomination based on counting `SDL_GetTicks()`.